generators/server/templates/src/main/docker/_app.dev.yml

```diff
@@ -72,5 +65,5 @@ services:
         <%_ } _%>
         <%_ if (applicationType == 'microservice') { _%>
         expose:
-            - "8080"
+            - "8081"
```

Comment:
Please forgive me, but I am confused by this... 
Why are you moving to port 8081, did I miss part of a thread somewhere?  And if so, why aren't you moving the [prod port](https://github.com/jhipster/generator-jhipster/pull/3085/files#diff-5e46f70be7afa0bd3a118c08d94e06b5L75) as well?